policy/resolver.go

```diff
@@ -1378,13 +1386,30 @@ func mquery2executionQuery(query queryLike, props map[string]*llx.Primitive, pro
 	return &res, dataChecksum, nil
 }
 
-func ensureControlJob(cache *frameworkResolverCache, jobs map[string]*ReportingJob, controlMrn string, framework *ResolvedFramework) *ReportingJob {
+func ensureControlJob(cache *frameworkResolverCache, jobs map[string]*ReportingJob, controlMrn string, framework *ResolvedFramework, frameworkGroupByControlMrn map[string]*FrameworkGroup) *ReportingJob {
 	uuid := cache.relativeChecksum(controlMrn)
 
 	if found, ok := jobs[uuid]; ok {
 		return found
 	}
 
+	// If we ignore this control, we have to transfer this info to the impact var,
+	// which is used to inform how to aggregate the results of all child jobs.
+	impact := &explorer.Impact{}
+	validUntil := ""
+	if frameworkGroup, ok := frameworkGroupByControlMrn[controlMrn]; ok {
+		if frameworkGroup.EndDate != 0 {
+			time.Unix(frameworkGroup.EndDate, 0).Format(time.RFC3339)
+		}
+		if frameworkGroup.Type == GroupType_IGNORED {
```

Comment:
if this is disabled, should it also have a changed impact?

Reply:
No, when it is disabled, it should not have a job at all. You can also see this in the tests. The one with a `DISABLE` group has less jobs than the `IGNORE` test. Both have the same framework otherwise.